icu4c/source/python/icutools/databuilder/filtration.py

```diff
@@ -56,6 +56,13 @@ def _file_to_file_stem(cls, file):
         limit = file.filename.rfind(".")
         return file.filename[start+1:limit]
 
+    @classmethod
+    def _file_to_subdir(cls, file):
```

Comment:
This is where my Python lets me down, but it seems odd to have a class method requiring an instance that doesn't use it (I guess it's copied from _file_to_file_stem. Also weird that there's no idiomatic util for this sort of thing.

Reply:
Do you mean in this PR or another. Feel free to do it in another since otherwise it messes up my approval (and you might want to merge today).

Reply:
I already made the `@staticmethod` change.